### PR TITLE
Danger: update success messaging for internal/external contributors

### PR DESCRIPTION
**Pull request by brbrr**
#### Changes proposed in this Pull Request:

* Update Danger success messaging

#### Testing instructions:

* Grab API token from Secret Stor by searching for `jetpackbot`
* Run `DANGER_GITHUB_API_TOKEN=$API_TOKEN ./node_modules/.bin/danger-pr $PR_URL`
* Run against PRs for internal & external contributors

#### Proposed changelog entry for your changes:

* none

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -1,42 +1,61 @@
...
+`**Thank you for the great PR description!**
+
+When this PR is ready for review, please apply the \`[Status] Needs Review\` label and if possible have someone from your team review the code. The Jetpack team will also review this PR and merge it to be included in the next release: _${ jetpackReleaseDate }_.
+Code freeze date: _${ codeFreezeDate }_`
+);
+		} else {
+			markdown(
+`**That's a great PR description, thank you so much for your effort!**
+
+Wondering what's will happen next? The Jetpack team will jump in and review this PR, and merge it once all the feedback will be addressed. Once it merged - your changes will be included in next Jetpack release: _${ jetpackReleaseDate }_`
```
Comment (jeherve) [inline dangerfile.js]: Maybe "once all the feedback has been addressed. Once the PR is merged, your changes"

Comment (jeherve) [inline dangerfile.js]: How about "in next Jetpack release, scheduled for _${ jetpackReleaseDate }_"

Comment (jeherve) [inline dangerfile.js]: Should we mention that we expect PRs to get 2 reviews somewhere here?

**Review by jeherve — CHANGES_REQUESTED** [review]
This is awesome, it should make the bot even more useful! 

I just had a few small remarks.

File: dangerfile.js (hunk 1 above)
Comment (brbrr) [reply dangerfile.js]: Is that makes any difference for the external contributor (since he can not assign reviewers)?

Comment (jeherve) [reply dangerfile.js]: That's a good point. Probably not. 👍

File: dangerfile.js [hunk 2]
```diff
@@ -1,42 +1,61 @@
...
+`**Thank you for the great PR description!**
+
+When this PR is ready for review, please apply the \`[Status] Needs Review\` label and if possible have someone from your team review the code. The Jetpack team will also review this PR and merge it to be included in next Jetpack release, scheduled for: _${ jetpackReleaseDate }_.
+Code freeze date: _${ codeFreezeDate }_`
+);
+		} else {
+			markdown(
+`**That's a great PR description, thank you so much for your effort!**
+
+Wondering what's will happen next? The Jetpack team will jump in and review this PR, and merge it once all the feedback has been addressed. Once the PR is merged, your changes will be included in next Jetpack release, scheduled for: _${ jetpackReleaseDate }_`
```
Comment (jeherve) [inline dangerfile.js]: "what will happen next" seems better.

File: dangerfile.js [hunk 3]
```diff
@@ -1,42 +1,61 @@
...
 	markdown( "This is automated check which relies on [`PULL_REQUEST_TEMPLATE`](https://github.com/Automattic/jetpack/blob/master/.github/PULL_REQUEST_TEMPLATE.md).We encourage you to follow that template as it helps Jetpack maintainers do their job. If you think 'Testing instructions' or 'Proposed changelog entry' are not needed for your PR - please explain why you think so. Thanks for cooperation :robot:" );
 } else {
-	markdown( "That's a great PR description, thank you so much for your effort!" );
+	schedule( async () => {
+		const internalContributor = await github.api.orgs.checkMembership( { org: 'Automattic', username: pr.user.login } );
+		if ( internalContributor ) {
+			markdown(
+`**Thank you for the great PR description!**
+
+When this PR is ready for review, please apply the \`[Status] Needs Review\` label and if possible have someone from your team review the code. The Jetpack team will also review this PR and merge it to be included in next Jetpack release, scheduled for: _${ jetpackReleaseDate }_.
```
Comment (jeherve) [inline dangerfile.js]: "in **the** next Jetpack release", maybe?

**Review by jeherve — DISMISSED** [review]
A few more minor changes.

**Review by jeherve — APPROVED** [review]
This seems to work well now, although I have to say I don't really understand the magic behind your calculation of the first Tuesday of the month :)

**Comment by kraftbj** [comment]
I'd like to hear more about the first Tuesday date too! Looks good, so merging.

**Pull request merged**